src/include/txn_inline.h

```diff
@@ -1043,9 +1045,25 @@ __wt_txn_read(
 
     /* If there's no visible update in the update chain or ondisk, check the history store file. */
     if (F_ISSET(S2C(session), WT_CONN_HS_OPEN) && !F_ISSET(session->dhandle, WT_DHANDLE_HS)) {
+        /*
```

Comment:
Do we need to change this check?
/* If there is no ondisk value, there can't be anything in the history store either. */
    if (cbt->ref->page->dsk == NULL || cbt->slot == UINT32_MAX) {
        cbt->upd_value->type = WT_UPDATE_TOMBSTONE;
        return (0);
    }

Reply:
@kommiharibabu Are you going to fix this?